src/app/visual/checkbox/checkbox-visual.component.html

```diff
@@ -1,79 +1,109 @@
 <sky-docs-demo-control-panel-theme
   (themeSettingsChange)="themeSettingsChange($event)"
-></sky-docs-demo-control-panel-theme>
+>
+</sky-docs-demo-control-panel-theme>
 
 <div id="screenshot-checkbox">
-  <sky-checkbox [checked]="true">
-    <sky-checkbox-label>Checked Checkbox</sky-checkbox-label>
+  <sky-checkbox
+    [checked]="true"
+  >
+    <sky-checkbox-label>
+      Checked Checkbox
+    </sky-checkbox-label>
   </sky-checkbox>
-  <sky-checkbox [checked]="false">
-    <sky-checkbox-label>Unchecked Checkbox</sky-checkbox-label>
+  <sky-checkbox
+    [checked]="false"
+  >
+    <sky-checkbox-label>
+      Unchecked Checkbox
+    </sky-checkbox-label>
   </sky-checkbox>
-  <sky-checkbox [checked]="false" [disabled]="true">
-    <sky-checkbox-label>Disabled Checkbox</sky-checkbox-label>
+  <sky-checkbox
+    [checked]="false"
+    [disabled]="true">
```

Comment:
Closing bracket `>` needs to be on a new line.

Reply:
weird... this is fixed locally, and it says i pushed my commit for that file. ill try pushing again

Reply:
nevermind, misread your comment. fixed in last commit!